ruma-serde/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 use serde::de::{Deserialize, IntoDeserializer};
 
 pub mod can_be_empty;
+pub mod canonical_json;
```

Comment:
Let's not make this public for now. The only thing defined in there is `value::CanonicalJsonValue`, which is also re-exported.